src/components/note-modal/NoteModal.jsx

```diff
@@ -1,69 +1,143 @@
-            notesDispatch({ type: "CLEAR_NOTES_INPUTS" });
-          }}
-        />
-        <h3>Title</h3>
-        <input
-          type="text"
-          className="input note-modal-input"
-          value={title}
-          required
-          onChange={(event) =>
-            notesDispatch({ type: "TITLE", payload: event.target.value })
-          }
-        />
-        <h3 className="m-t-1rem">Description</h3>
-        <textarea
-          className="input note-modal-input"
-          rows="4"
-          value={description}
-          required
-          onChange={(event) =>
-            notesDispatch({ type: "DESCRIPTION", payload: event.target.value })
-          }
-        />
-        <div className="text-center">
-          {!noteId ? (
-            <button
-              className="btn btn-primary m-t-1rem"
-              onClick={(event) =>
-                addToNotes(event, notesState, notesDispatch, toastDispatch)
+      <div className={`modal note-modal-container ${backgroundColor}`}>
+        <form>
+          <AiFillCloseCircle
+            className="card-close-pos card-pos-abs color-green"
+            onClick={() => {
+              notesDispatch({ type: "DISPLAY_MODAL", payload: false });
+              notesDispatch({ type: "CLEAR_NOTES_INPUTS" });
+            }}
+          />
+          <h3>Title</h3>
+          <label htmlFor="title">
+            <input
+              type="text"
+              id="title"
+              className="input note-modal-input"
+              value={title}
+              required
+              onChange={(event) =>
+                notesDispatch({ type: "TITLE", payload: event.target.value })
               }
-            >
-              Add note
-            </button>
-          ) : (
-            <button
-              className="btn btn-primary m-t-1rem"
-              onClick={(event) =>
-                editNote(event, notesState, notesDispatch, toastDispatch)
+            />
+          </label>
+          <h3 className="m-t-1rem">Labels</h3>
+          <div className="flex flex-start gap-1rem">
+            <label htmlFor="office" className="cursor">
+              <input
+                type="radio"
+                name="label"
+                id="office"
+                checked={labelForNote === "office"}
+                onChange={() =>
+                  notesDispatch({ type: "LABEL_FOR_NOTE", payload: "office" })
+                }
+              />
+              Office
+            </label>
+            <label htmlFor="food" className="cursor">
+              <input
+                type="radio"
+                name="label"
+                id="food"
+                checked={labelForNote === "food"}
+                onChange={() =>
+                  notesDispatch({ type: "LABEL_FOR_NOTE", payload: "food" })
+                }
+              />
+              Food
+            </label>
+            <label htmlFor="home" className="cursor">
```

Comment:
suggestion: You can move this tag code to separate components so you can reuse them without repeating code if number of tag increases